input/httplisten/httplisten_test.go

```diff
@@ -101,3 +101,47 @@ input:
 		assert.Equal(map[string]interface{}{"foo": "bar"}, event.Extra)
 	}
 }
+
+func Test_input_httpslisten2_module(t *testing.T) {
+	assert := assert.New(t)
+	assert.NotNil(assert)
+	require := require.New(t)
+	require.NotNil(require)
+
+	ctx := context.Background()
+	conf, err := config.LoadFromYAML([]byte(strings.TrimSpace(`
+debugch: true
+input:
+  - type: httplisten
+    address: "127.0.0.1:8989"
```

Comment:
You listen the same port with `Test_input_httpslisten_module`, and listen error doesn't return. So you pass the unit test...Q_Q

Reply:
good catch! though it'll not happen in real world.
I was thinking this feature should have service close API so that we don't need to listen on different ports in UT.